### PR TITLE
Resnext translation

### PR DESCRIPTION
**Pull request by JunyongKang**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #(숫자)

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 모델 소개를 번역하는 기여
- [ ] 공식 허브의 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._

### CODE REVIEW

File: pytorch_vision_resnext.md [hunk 1]
```diff
@@ -83,18 +83,18 @@ for i in range(top5_prob.size(0)):
     print(categories[top5_catid[i]], top5_prob[i].item())
 ```
 
-### Model Description
+### 모델 설명
 
 Resnext 모델은 논문 [Aggregated Residual Transformations for Deep Neural Networks]에서 제안되었습니다. (https://arxiv.org/abs/1611.05431).
 이중 두가지 버전의 모델 성능은 아래와 같습니다. 각 모델의 레이어 개수는 각 50, 101개입니다.
 resnet50과 resnext50의 아키텍처 차이는 논문의 Table 1을 참고하십시오.
 ImageNet 데이터셋에 대한 사전훈련된 모델의 에러(성능)은 아래 표와 같습니다.
```
Comment (petanerd) [inline pytorch_vision_resnext.md:91]: `모델 설명`의 본문에서는 '에러', 해당 내용을 가르키는 아래 표에서는 '오류'로 작성 되어있습니다.
외래어 표기법 등에 따라 둘 중에 하나로 통일하는 것이 좋지 않을까 생각 됩니다 :)

File: pytorch_vision_resnext.md [hunk 2]
```diff
@@ -1,11 +1,4 @@
----
-layout: hub_detail
-background-class: hub-background
-body-class: hub
-title: ResNext
-summary: Next generation ResNets, more efficient and accurate
```
Comment (mindongjun) [inline pytorch_vision_resnext.md:6]: torch hub 페이지를 보면 `summary`부분 내용이 최상단에 표시되는데 한줄 요약으로 모델에 대한 간결한 설명이 되어있어, 이부분도 번역이 되면 좋지 않을까 생각됩니다.

**Review by petanerd — APPROVED** [review]

File: pytorch_vision_resnext.md [hunk 3]
```diff
@@ -25,23 +18,20 @@ model = torch.hub.load('pytorch/vision:v0.10.0', 'resnext50_32x4d', pretrained=T
...
 
-All pre-trained models expect input images normalized in the same way,
-i.e. mini-batches of 3-channel RGB images of shape `(3 x H x W)`, where `H` and `W` are expected to be at least `224`.
-The images have to be loaded in to a range of `[0, 1]` and then normalized using `mean = [0.485, 0.456, 0.406]`
-and `std = [0.229, 0.224, 0.225]`.
-
-Here's a sample execution.
+모든 사전훈련된 모델은 입력 이미지가 같은 방식으로 정규화되었다고 가정합니다.
+즉, 미니배치(mini-batch)의 3채널 RGB 이미지들은 `(3 x H x W)`의 shape을 가지며, `H`와 `W`는 최소 `224`이상이어야 하며, 각 이미지들은 `[0, 1]`의 범위에서 로드되어야 하며, 그 다음 `mean = [0.485, 0.456, 0.406]` 과 `std = [0.229, 0.224, 0.225]`를 이용해 정규화되어야 합니다. 
+아래 예시 코드가 있습니다.
```
Comment (yh00214) [inline pytorch_vision_resnext.md]: 1. 22번째 줄 문장이 원문에서 두 문장으로 된 것을 합친 것으로 보입니다. TRANSLATION_GUIDE에 의거하여 문장 단위는 가급적 유지할 수 있도록, 문장을 두 문장으로 나누는 편이 좋을 것 같습니다.
2. 22번째 줄의 '각 이미지들은 `[0, 1]`의 범위에서 로드되어야 하며'의 경우, 기존 다른 문서의 PR에서 로드되어야 한다는 말의 의미를 설명하는 게 좋겠단 리뷰를 받아들여 해당 부분의 설명을 추가한 사례가 있습니다. 현재 문서에도 해당 내용을 추가해도 좋을 것 같다고 생각합니다.
https://github.com/PyTorchKorea/hub-kr/pull/8/commits/df0adb6e981ae782c9df17b0f962a049a9c9628e

**Review by yh00214 — COMMENTED** [review]
수고하셨습니다!
